node-definitions/cisco/NXOSv 9500 128 ports/readme.md

```diff
@@ -0,0 +1,18 @@
+# NXOSv 9500 with 128 ports
+This directory contains the following node definition:
+
+*`nxosv9500-148.yaml` - Nexus 9500v with 128 ports node definition
```

Comment:
This says 128 switch ports, but your node def talks about 148.  Ultimately, I do see that it is 148 (64 + 16 + 32 + 36).  So it's probably just this typo that needs to be fixed.